test/performance/odp_l2fwd.c

```diff
@@ -108,6 +108,7 @@ typedef struct {
 	int error_check;        /**< Check packet errors */
 	int sched_mode;         /**< Scheduler mode */
 	int num_groups;         /**< Number of scheduling groups */
+	int verbose;		/**< Verbose output */
 } appl_args_t;
```

Comment:
It's not obvious what these changes have to do with the stated purpose of this PR. This looks like a separate PR altogether.

Reply:
I used this option to check the frame sizes. E.g. dpdk pktio returned IP MTU size (1500), not max ethernet frame size (1514). This an easy way to check it on various pktios as l2fwd takes interface as a command line parameter.